techniques/flash/FLASH34.html

```diff
@@ -26,20 +26,20 @@
     				is running. </p>
          <p>Other assistive technology tools, including screen magnifiers,
     				or tools not used as assistive technologies may also utilize MSAA in
-    				ways that result in <code class="prop">Accessibility.active</code> being set to <code class="propval">true</code>. </p>
+    				ways that result in <code>Accessibility.active</code> being set to <code>true</code>. </p>
       </div>
    </section><section id="examples"><h2>Examples</h2>
       <section class="example">
          <h3>A SoundHandler class</h3>
          
             <p>A class called SoundHandler is created which automatically starts
-    						playing an MP3 file only when <code class="prop">Accessibility.active</code> is set to <code class="propval">false</code>.
-    						Note that this example also checks the <code class="prop">flash.system.Capabilities.hasAccessibility</code>
+    						playing an MP3 file only when <code>Accessibility.active</code> is set to <code>false</code>.
+    						Note that this example also checks the <code>flash.system.Capabilities.hasAccessibility</code>
     						property. This property does not check whether a screen reader is running,
     						but instead indicates whether the Flash Player is running in an environment
     						that supports MSAA (which basically means the Windows operating system). </p>
          
-         <pre xml:space="preserve">package wcagSamples {
+         <pre>package wcagSamples {
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`